analyses/malawi/dryspells_trigger/exploration/plot_resolution_comparison.md

```diff
@@ -0,0 +1,74 @@
+## Plot ARC2 and CHIRPS resolution
+
+Simple notebook to investigate the resolution of ARC2 and CHIRPS data when compared against various admin levels in Malawi.
+
+```python
+from pathlib import Path
+import sys
+import os
+
+import rioxarray
+from shapely.geometry import mapping
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+mpl.rcParams['figure.dpi'] = 300
+```
+
+Set the config values and parameters.
+
+```python
+path_mod = f"{Path(os.path.dirname(os.path.abspath(''))).parents[2]}/"
+sys.path.append(path_mod)
+from src.indicators.drought.config import Config
+
+config = Config()
+parameters = config.parameters('malawi')
+COUNTRY_ISO3 = parameters["iso3_code"]
+DATA_DIR = Path(config.DATA_DIR)
+
+RAW_DIR =  DATA_DIR / config.PUBLIC_DIR / config.RAW_DIR / COUNTRY_ISO3
+ARC2_DIR = DATA_DIR / config.PUBLIC_DIR / 'exploration' / COUNTRY_ISO3 / 'arc2'
+ARC2_FILEPATH = ARC2_DIR / "arc2_20002020_approxmwi.nc"
+
+ADM1_SHP = RAW_DIR / config.SHAPEFILE_DIR / parameters['path_admin1_shp']
+ADM2_SHP = RAW_DIR / config.SHAPEFILE_DIR / parameters['path_admin2_shp']
+ADM3_SHP = RAW_DIR / config.SHAPEFILE_DIR / parameters['path_admin3_shp']
+
+PLOT_DIR = DATA_DIR / 'public' / 'processed' / 'mwi' / 'plots' / 'dry_spells' / 'arc2'
+```
+
+Read in data.
+
+```python
+# Admin boundaries
+df_adm3 = gpd.read_file(ADM1_SHP)
```

Comment:
All of the code and outputs look good, I think just the adm3 and adm1 files got switched around so adm1 is being mapped instead of adm3 down below.

Reply:
Good catch!